### ISSUE TITLE
Missing FAQ template?

### ISSUE THREAD

**Issue by greenbagels** [akatch/hammerbros#21]
git hash: 63f30ef881ae534b7fddae2edcba7beea35dcff8
hugo version: v0.92.1-85E2E862+extended

When trying to serve the site from 63f30ef881ae534b7fddae2edcba7beea35dcff8, i get the following error:
```
Start building sites …
hugo v0.92.1-85E2E862+extended linux/amd64 BuildDate=2022-01-27T11:44:41Z VendorInfo=gohugoio
Error: Error building site: "/home/spervaiz/code/hammerbros/content/pages/curriculum/storyboards.md:9:1": failed to extract shortcode: template for shortcode "faq" not found
Built in 1055 ms
```
Do you have some unstaged template files? or am I missing something?

**Comment by akatch** [comment]
Thank you for reporting, this should be fixed with the abovementioned PR.

**Comment by akatch** [comment]
Fixed with #23
